packages/next-remoter/src/composable/CustomAgentModelProvider.ts

```
@@ -48,8 +52,13 @@ export class CustomAgentModelProvider extends BaseModelProvider {
   }
 
   async chatStream(request: ChatCompletionRequest, handler: StreamHandler): Promise<void> {
+    debugger
+    const lastUserMsg = request.messages.findLast((msg) => msg.role === 'user' && msg.content !== '')
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)